docker-compose-expose-mysql.yml

```diff
@@ -0,0 +1,8 @@
+# uses port 80 instead of 3000
+# to use this create an .env file with the following content:
+# COMPOSE_FILE=docker-compose.yml:docker-compose-port-80.yml
```

Comment:
This comment seems out of date

Reply:
Oops, copy-pasta from the chome port override file.  I'll fix it.

Reply:
Do you mind if I fix that in the rails 4.1 -> 4.2 branch I'm in now so I don't have to rebase?